bin/test-all-exercises

```diff
@@ -5,7 +5,9 @@ exercises=$(dirname $(dirname $0))/exercises
 status=0
 failed_exercises=""
 
-for d in $exercises/*/; do
+# TODO: To be confirmed: Would use this line when there are any concept exercises
+#for d in $exercises/{concept,practice}/*/; do
```

Comment:
and you would think "wouldn't it be nice to just use this line now and not have to change it later" ? But, since `exercises/concept` doesn't exist, the glob expansion fails, even though `exercises/practice` does exist. So, no, we cannot use this line before a concept exercise does in fact exist.